source/chapters/1/meteorSnacks.md

```diff
@@ -148,49 +158,47 @@ Meteor.methods({
 
 ## Assets
 
-http://docs.meteor.com/api/assets.html
-
-Go ahead, put something in "/private/some_folder/test.txt":
+The best documentation for assets is available [here](http://docs.meteor.com/api/assets.html), at the official Meteor website!
 
+Let's go ahead and put something in "/private/some_folder/test.txt" by running this in our Meteor shell:
 ```
-// meteor shell
 Assets.getText('/some_folder/test.txt')
 ```
 
-You would use this when for example, you have a business, logic-packed csv, or .xls file.
-Or you may have a JSON with car models and makes. 
-The idea is that you can have any type of file, even binary, that you can use privately on the server.
+You would use this when, for example, you have a business, logic-packed csv, or an .xls file that you need to work with.
+Or you may have a JSON with car models. 
+The idea is that you can use any type of file, even binary, that you can use privately on the server where the application is deployed on.
 
 ## Meteor Settings
 
-https://docs.meteor.com/api/core.html#Meteor-settings
-
+Again, the guys and girls at [Meteor](https://docs.meteor.com/api/core.html#Meteor-settings) explain this subject in the best way.
```

Comment:
I would remove the guys and girls, maybe: Again, the deevelopment team ...explain...